src/main/java/org/example/tokpik_be/tag/service/TopicTagService.java

```diff
@@ -0,0 +1,50 @@
+package org.example.tokpik_be.tag.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.example.tokpik_be.exception.GeneralException;
+import org.example.tokpik_be.exception.UserException;
+import org.example.tokpik_be.tag.domain.TopicTag;
+import org.example.tokpik_be.tag.dto.response.UserTopicTagResponse;
+import org.example.tokpik_be.tag.entity.UserTopicTag;
+import org.example.tokpik_be.tag.repository.UserTopicTagRepository;
+import org.example.tokpik_be.user.domain.User;
+import org.example.tokpik_be.user.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class TopicTagService {
+
+    private final UserTopicTagRepository userTopicTagRepository;
+
+    private final UserRepository userRepository;
+
+    public UserTopicTagResponse getUserTopicTags(long userId) {
+
+        User user = findById(userId);
+
+        List<UserTopicTag> userTopicTags = userTopicTagRepository.findByUserId(user.getId());
+
+        List<UserTopicTagResponse.TopicTagDTO> topicTagDTOList = userTopicTags.stream()
+            .map(userTopicTag -> {
+                TopicTag topicTag = userTopicTag.getTopicTag();
+                return new UserTopicTagResponse.TopicTagDTO(
+                    topicTag.getId(),
+                    topicTag.getContent()
+                );
+            })
+            .collect(Collectors.toList());
```

Comment:
```suggestion
            .toList();
```
Java 17에서 제공하는 `toList()`로 더 간결하게 구현 가능합니다.